src/fish.h

```diff
@@ -10,13 +10,15 @@
 #define INITIAL_HUNGER 59.7 * 30  // 30 seconds
 #define COIN_FULLNESS_LIMIT 3 
 #define FISH_MOVE_INTERVAL 60     // Number of frames before a fish changes direction
+#define HUNGER_THRESHOLD 59.7 * 10  // Fish starts seeking food when the hunger timer is below this - about 10 seconds
```

Comment:
```suggestion
#define HUNGER_THRESHOLD (59.7 * 10)  // Fish starts seeking food when the hunger timer is below this - about 10 seconds
```
Without parenthesis you can end up doing some strange things because of the order of operations. 

Consider:
```c
if (HUNGER_THRESHOLD > 5)
... becomes
if (59.7 * 10 > 5)
```

It may still be correct but it's ambigious

Reply:
Good catch, have made that change. Cheers.